configure.py

```diff
@@ -34,11 +34,12 @@ def _VALID_BAZEL_VERSION(tf_version):
         'refering to the previous COMMIT to compile properly by themselves.')
     return "0.26.1"
   elif tf_version >= "2.0.0":
+    target_bazel = "4.1.0" if is_macos() and is_arm64() else "3.7.2"
     logging.info(
         'To ensure code compatibility with Bazel rules_foreign_cc component, '
         'we specify Bazel version greater than 3.7.2 '
```

Comment:
3.7.2 is not accuracy, we should use `target_bazel`

Reply:
I think this line will assign `target_bazel` value "3.7.2" if the platform isn't macos or arm64.
Shall we initialize target_bazel at the first line, and assign the value to it according to the condition?
Changed to the following,
`
if is_macos() and is_arm64():
    target_bazel = "4.1.0"
    logging.warn(
        'Only Bazel version greater than 4.1.0 supports macOS arm64 platform.')
    return target_bazel
  elif tf_version < "2.0.0":
    target_bazel = "0.26.1"
    logging.warn(
        'There is only limited support for TensorFlow under version 2.0.0 '
        'because its Bazel version, and requiring users to make some Bazel script changes '
        'refering to the previous COMMIT to compile properly by themselves.')
    return target_bazel
  elif tf_version >= "2.0.0":
    target_bazel = "3.7.2"
    logging.info(
        'To ensure code compatibility with Bazel rules_foreign_cc component, '
        'we specify Bazel version greater than 3.7.2 '
        'for Tensorflow versions greater than 2.0.0.')
    return target_bazel
  else:
    raise ValueError('Unsupport TensorFlow version {}.'.format(tf_version))
`

A little ugly though :(

Reply:
It's OK.
If I write I'll replace 4.1.0 in logging string by `.format(target_bazel)`, I don't like express single thing by using two different form, but your code is OK, piece of cake:)